### PR TITLE
ADR-796: Changes for duty suspended deliveries outside of the UK

### PR DESCRIPTION
**Pull request by sini-george**
These files changed are for duty suspended deliveries outside of the UK - ADR-796

### CODE REVIEW

**Review by Ewan-James-Donovan — COMMENTED** [review]
Just some suggestions :) Yash will need to take a look, too

File: app/views/DeclareDutySuspendedDeliveriesOutsideUkView.scala.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      @if(form.errors.nonEmpty) {
+          @govukErrorSummary(ErrorSummaryViewModel(form))
+      }
+
+      @sectionHeading(
+          id = "declare-duty-suspended-deliveries-outside-uk-section",
+          text = messages("section.alcoholDutyReturn"),
+      )
+
+        <h1 class="govuk-heading-l">@messages("declareDutySuspendedDeliveriesOutsideUk.heading")</h1>
```
[the remark this answers is not in the document]
Comment (yasaracar) [reply app/views/DeclareDutySuspendedDeliveriesOutsideUkView.scala.html:46]: agreed

File: app/views/DeclareDutySuspendedDeliveriesOutsideUkView.scala.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+          <p class="govuk-body">
+              @messages("declareDutySuspendedDeliveriesOutsideUk.p1")
+          </p>
+          <p class="govuk-hint">
```
[the remark this answers is not in the document]
Comment (yasaracar) [reply app/views/DeclareDutySuspendedDeliveriesOutsideUkView.scala.html:51]: agreed

File: app/views/DeclareDutySuspendedDeliveriesOutsideUkView.scala.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+      @govukInput(
+          InputViewModel(
+          field = form("declare-duty-suspended-deliveries-outside-uk-input"),
+          label = LabelViewModel(messages("declareDutySuspendedDeliveriesOutsideUk.heading")).asVisuallyHidden(),
+          )
+         .asNumeric()
+         .withWidth(Fixed10)
+         .withSuffix(PrefixOrSuffix(content = messages("declareDutySuspendedDeliveriesOutsideUk.inputSuffix")))
+
```
[the remark this answers is not in the document]
Comment (yasaracar) [reply app/views/DeclareDutySuspendedDeliveriesOutsideUkView.scala.html:63]: agreed

File: conf/messages.en [hunk 4]
```diff
@@ -108,3 +108,17 @@ declareDutySuspendedDeliveriesQuestion.title = Do you need to declare alcohol yo
...
+declareDutySuspendedDeliveriesOutsideUk.error.nonNumeric = How much you have delivered duty suspended must be a number
+declareDutySuspendedDeliveriesOutsideUk.error.outOfRange = DeclareDutySuspendedDeliveriesOutsideUk must be between {0} and {1}
+declareDutySuspendedDeliveriesOutsideUk.change.hidden = DeclareDutySuspendedDeliveriesOutsideUk
+declareDutySuspendedDeliveriesOutsideUk.error.minimumRequired = How much you have delivered duty suspended must be 0.01 or more
+declareDutySuspendedDeliveriesOutsideUk.error.maximumRequired = How much you have delivered duty suspended must be 999999999.99 or less
+declareDutySuspendedDeliveriesOutsideUk.error.twoDecimalPlaces = How much you have delivered duty suspended must be a number to 2 decimal places
+declareDutySuspendedDeliveriesOutsideUk.inputSuffix = litres
+declareDutySuspendedDeliveriesOutsideUk.p1 = This includes to sites in EU countries, sites in countries outside of the EU and ship stores. This will not change your duty calculation.
+declareDutySuspendedDeliveriesOutsideUk.p2 = Give your answers in litres, to 2 decimal places.
```
[the remark this answers is not in the document]
Comment (yasaracar) [reply conf/messages.en:124]: agreed

File: conf/messages.en [hunk 5]
```diff
@@ -108,3 +108,17 @@ declareDutySuspendedDeliveriesQuestion.title = Do you need to declare alcohol yo
 declareDutySuspendedDeliveriesQuestion.heading = Do you need to declare alcohol you delivered or received duty suspended?
 declareDutySuspendedDeliveriesQuestion.error.required = Select yes if you need to declare any Duty Suspended Deliveries
 declareDutySuspendedDeliveriesQuestion.p1 = For example, because you delivered it to a duty suspended warehouse.
+
+declareDutySuspendedDeliveriesOutsideUk.title = How much have you delivered duty suspended outside of the UK?
+declareDutySuspendedDeliveriesOutsideUk.heading = How much have you delivered duty suspended outside of the UK?
+declareDutySuspendedDeliveriesOutsideUk.checkYourAnswersLabel = DeclareDutySuspendedDeliveriesOutsideUk
+declareDutySuspendedDeliveriesOutsideUk.error.required = Enter how much you have delivered duty suspended
+declareDutySuspendedDeliveriesOutsideUk.error.nonNumeric = How much you have delivered duty suspended must be a number
+declareDutySuspendedDeliveriesOutsideUk.error.outOfRange = DeclareDutySuspendedDeliveriesOutsideUk must be between {0} and {1}
```
Comment (yasaracar) [inline conf/messages.en:117]: We don't seem to be using this error message, can you remove this line if that is the case?

**Review by yasaracar — CHANGES_REQUESTED** [review]

**Review by yasaracar — CHANGES_REQUESTED** [review]

**Review by yasaracar — APPROVED** [review]

**Review by Ewan-James-Donovan — APPROVED** [review]

**Pull request merged**